src/widget/wcoverartmenu.cpp

```diff
@@ -82,8 +86,38 @@ void WCoverArtMenu::slotChange() {
     coverInfo.source = CoverInfo::USER_SELECTED;
     coverInfo.coverLocation = selectedCoverPath;
     coverInfo.setImage(image);
-    qDebug() << "WCoverArtMenu::slotChange emit" << coverInfo;
-    emit coverInfoSelected(coverInfo);
+
+    QString coverArtCopyFilePath =
+            fileInfo.absoluteFilePath().left(
+                    fileInfo.absoluteFilePath().lastIndexOf('.') + 1) +
```

Comment:
can we replace this with completeBaseName()?

Reply:
This can be replaced with the absolutePath() + completeBaseName() + extension.

Reply:
I also need to add '/' between the path and the name, and also '.' between the name and extension. I guess that wouldn't be ok between the different OS. 

If that's the case, it is better to stay like this, this basically just removes the trackfilepath.'trackExtension' and then adds the selected extension. I have looked for it but couldn't find. Is there a better way to do this without errors in different OS?

Reply:
OK